spynl/main/validation.py

```
@@ -120,8 +120,7 @@ def apply_schema(data, schema_name):
                                "against a schema", data)
             jsonschema.validate(data, schema)
         except (KeyError, jsonschema.ValidationError) as val_exc:
-            raise InvalidResponse(_('invalid-response',
-                                    default=val_exc.message))
+            raise InvalidResponse(_(val_exc.message))
```

It should be `raise InvalidResponse(val_exc.message)` without the _()